include/contractor/graph_contractor.hpp

```diff
@@ -47,7 +47,7 @@ class GraphContractor
                            bool shortcut,
                            bool forward,
                            bool backward)
-            : weight(weight), id(id), originalEdges(std::min((unsigned)1 << 28, original_edges)),
+            : weight(weight), id(id), originalEdges(std::min((1u << 28) - 1u, original_edges)),
```

Comment:
This std::min seems to be for prevention of overflow, since originalEdges became a 28 bit field.
But if you assign 1u<<28 you will acually assign 0 ...